index.js

```diff
@@ -0,0 +1,29 @@
+function calculate(calculationSteps) {
+  // let operators = ['+', '-', '*', '/']
+
+  for (let i = 0; i < calculationSteps.length; i++) {
+    if (calculationSteps[i] === null) {
+      return 0
```

Comment:
if any value equals null... we need to use it as a 0 instead.

we do not return 0